Enrich_methods/example.py

```diff
@@ -0,0 +1,210 @@
+
+enriched_mat = mdbt.Material({30060000: 0.6, 30070000: 0.4})
+atom_frac_enriched = mdbt.enrich(
+    pure["Li2TiO3nat"]["atom_frac"], 30000000, enriched_mat.to_atom_frac()
+)
+# I would need to call enrich() each time I wanted to enrich a different element
+mat_input3 = {
+    "atom_frac": atom_frac_enriched,
+    "density": 3.42,
+    "citation": "HernandezFusEngDes_2018",
+}
+material3 = mdbt.make_mat_from_atom(**mat_input3)
+# From here you can either add it to the input library or call it in def hcpb_pin_bw
+
+
+################################### Method 4
+
+# In this method make_mat_from_atom can call the enrich function
+# I can enrich multiple isotopes at the same time with this method
+
+pure["Li2TiO3nat"]["mass_enrichment"] = {30000000: {30060000: 0.6, 30070000: 0.4}}
+# I can enrich multiple isotopes at the same time with this method
+material4 = mdbt.make_mat_from_atom(**pure["Li2TiO3nat"])
+mat_lib_test["material4_Li2TiO3"] = material4
+
+# Can also be either added to the input library or call it in def hcpb_pin_bw
+# Inputting to the library allows for the mix
+pure["Li4SiO4nat"]["mass_enrichment"] = {30000000: {30060000: 0.6, 30070000: 0.4}}
+mat_lib_test["material4_Li3SiO4"] = mdbt.make_mat_from_atom(**pure["Li4SiO4nat"])
+
+# You can theoretically enrich the material with one line
+#mat_lib_test["material4_Li3SiO4"] = mdbt.make_mat_from_atom(**{**pure.setdefault("Li4SiO4nat", {}), "mass_enrichment": {30000000: {30060000: 0.6, 30070000: 0.4}}}})
+
+mixed_materials = {}
+
+mixed_materials["hcpb_enrich_method_4"] = {
+    "vol_fracs": {
+        "EUROFER97": 0.418,
+        "material4_Li3SiO4": 0.065,
+        "material4_Li2TiO3": 0.035,
+        "HeT410P80": 0.482,
+    },
+    "mixture_citation": "ZhouEnergies_2023 and ???",
+}
+
+# If added to the input library you can quite easily combine it with Method 5 but not need to call the mass enrichment function
+
+################################### Method 5
+
+# In this method make_mat_from_atom can be called by mix_by_volume, and make_mat_from_atom can call the the enrich function
+# Using Method I'm suggesting, one issue is that it requires the mdbt to import the library that has the materials before they have been created
+# With this method I can enrich multiple materials and multiple elements in those materials at the same time.
+# It also makes it trivial to add other materials and enrich them
+# It would be also fairly easy to add some of the other features such as adjusting the density of materials in this format
+
+
+mixed_materials["hcpb_enrich_method_5"] = {
+    "vol_fracs": {
+        "EUROFER97": 0.418,
+        "Li4SiO4nat": 0.065,
+        "Li2TiO3nat": 0.035,
+        "HeT410P80": 0.482,
+    },
+    "mixture_citation": "ZhouEnergies_2023 and ???",
+    "mass_enrichment": {
+        "Li2TiO3nat": {30000000: {30060000: 0.6, 30070000: 0.4}},
+        "Li4SiO4nat": {30000000: {30060000: 0.6, 30070000: 0.4}},
+    },
```

Comment:
Related to my comment below... what if we had a method that just took a material library and this dictionary and made a temporary library with enriched materials, and then we let `mix_by_volume` take multiple libraries...?